scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -26,6 +26,12 @@ namespace <%= namespace %> {
             }
         }
 
+        public List<UserError> UserErrors {
```

Comment:
I guess you could also see it as part of the Cart state. Like the cart is in an invalid state and this is why.